README.md

```diff
@@ -16,17 +16,17 @@ Finally, **microlinkjs** is powered by [Microlink API](https://docs.microlink.io
 
 ## Integration
 
-?> Did not find your connector? let's [open an issue](https://github.com/microlinkhq/microlinkjs/issues) for create it.
+Can't find your connector? Let's [open an issue](https://github.com/microlinkhq/microlinkjs/issues) to create it.
```

Comment:
Hey, need to put `?>` at the beginning.

This is part of https://docsify.js.org/ for rendering  help messages

<img width="801" alt="screen shot 2017-12-14 at 03 05 28" src="https://user-images.githubusercontent.com/2096101/33971964-a8a44ac8-e07b-11e7-8131-b0f4c731b9db.png">

Reply:
🤦‍♂️ 

Aha, will just have to stick with the fact it's on the GH readme 😅